packages/prisma/migrations/20220803090845_migrate_daily_event_reference_to_booking_reference/migration.sql

```diff
@@ -0,0 +1,4 @@
+UPDATE "BookingReference"
+SET "meetingUrl" = "dailyurl", "meetingPassword" = "dailytoken"
+FROM "DailyEventReference"
+WHERE "DailyEventReference"."bookingId" = "BookingReference"."bookingId" AND "BookingReference"."type" = 'daily_video'
```

Comment:
Will this migration move everything from `DailyEventReference` to `BookingReference` I'm having a hard time understanding what this query does exactly.

Reply:
From my understanding of how the system deals with booking and observation from DB, the system has two records for each daily video call booking, one inside a "BookingReference" with "type" = 'daily_video' and "meetingUl" eq `/video/{uid}` and one in "DailyEventReference" with real daily call url and token.

So by this migration, we will update all "BookingReference" records with "type" = 'daily_video' and related to the same booking Id with info from "DailyEventReference".

If, for historical reasons, the system could have "DailyEventReference" records without identical "BookingReference" then we need to write one more SQL query.